docs/reference/units.ipynb

```diff
@@ -448,6 +471,192 @@
    "source": [
     "sc.to_unit(1.0 * sc.Unit('s'), 'm')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Unit Aliases\n",
+    "\n",
+    "It is possible to define custom aliases for units.\n",
+    "This can be used to\n",
+    "\n",
+    "- guide string formatting to prefer a certain unit, e.g. angstrom over nm\n",
+    "- define domain specific units that can be expressed in terms of other units to\n",
+    "  - guide string formatting\n",
+    "  - construct units from strings with custom names\n",
+    "  \n",
+    "## Prioritizing Units in String Formatting\n",
+    "\n",
+    "When dealing with crystals or molecules, it is often convenient to use angstrom as a unit.\n",
+    "But by default, string formatting tends to prefer different bases in composite units.\n",
```

Comment:
MAybe we should make clear that 'angstrom' on its own works well and nothing has to be done... only int the combination with other units it may cause trouble?